content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,199 @@
+- Resource limits (disk, memory, cpu count) are supported in the same fashion as processes 
+on the host.
+- Both Named pipe mounts and Unix domain sockets are **not** currently supported and should instead be accessed via their 
+path on the host (e.g. \\\\.\\pipe\\\*)
+
+## {{% heading "prerequisites" %}}
+
+To enable HostProcess containers while in Alpha you need to pass the following feature gate flag to **kubelet** and **kube-apiserver**. 
+See [Features Gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#overview) documentation for more details.
+
+```
+--feature-gates=WindowsHostProcessContainers=true
+```
+
+You can use the latest version of Containerd (v1.5.4+) with the following settings using the containerd v2 configuration.  
+Add these annotations to any runtime configurations were you wish to enable the HostProcess Container feature.
+
+
+```
+[plugins]
+  [plugins."io.containerd.grpc.v1.cri"]
+    [plugins."io.containerd.grpc.v1.cri".containerd]
+      [plugins."io.containerd.grpc.v1.cri".containerd.default_runtime]
+        container_annotations = ["microsoft.com/hostprocess-container"]
+        pod_annotations = ["microsoft.com/hostprocess-container"]
+    [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
+      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runhcs-wcow-process]
+        container_annotations = ["microsoft.com/hostprocess-container"]
+        pod_annotations = ["microsoft.com/hostprocess-container"]
+```
+
+The current versions of containerd ship with a version of hcsshim that does not have support. You will need to build 
+a version of hcsshim from the main branch following the [instructions in hcsshim](https://github.com/Microsoft/hcsshim/#containerd-shim). 
+Once the containerd shim is built you can replace the file in your contianerd installation.  For example if you followed the instructions
+ to [install containerd](https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd) replace the
+  `containerd-shim-runhcs-v1.exe` is installed at `$Env:ProgramFiles\containerd` with the newly built shim.
+
+We are working to improve this process by enabling HostProcess support directly in 
+[containerd via the CRI api](https://github.com/containerd/containerd/pull/5131) as well as shipping a version of the containerd shim
+ for Windows with support direclty via containerd releases.
+
+## Creating a HostProcess Pod
+
+### Security Policy Config
+Enabling a Windows HostProcess pod simply requires setting the right configurations in the pod security configuration. 
+Many of the policies in the [Pod Security Standards](/docs/concepts/security/pod-security-standards) do not apply to 
+HostProcess containers (and Windows in general) due to architectural differences between Windows and Linux. As such, 
+here are the policy configurations required to enable HostProcess pods:
```

Comment:
This confuses me a bit.  I don't know how this fits into the https://kubernetes.io/docs/concepts/security/pod-security-standards/#policy-types.  Would these be additional fields in the `Privileged ` type?

Reply:
Updated to make this more clear.